src/components/forms/validations.jsx

```diff
@@ -28,8 +28,8 @@ module.exports.validationHOCFactory = defaultValidationErrors => (Component => {
         <Component
             validationErrors={defaults(
                 {},
```

Comment:
So there's a bug fix here, I think

the order of these params was wrong, I think... it kept you from being able to override the default message for formsy's isDefaultRequiredValue

@rschamp, LMK if you think it would make sense to make this a separate PR instead. (We'd lose the custom messages here until that was merged in.)

Reply:
This new order looks correct to me. I think it's fine to include it here.